src/App.tsx

```diff
@@ -312,10 +315,17 @@ export default function App() {
       const response = await getUser();
       if (response.requestSucceeded) {
         setUser(response.user ?? undefined);
+        if (response.user && !userState?.gtmInstrumented && (window as any).INIT_GTAG) {
```

Comment:
using recoil to keep state of whether this was instrumented.. we are loading the user multiple times depending on the view.. we don't want to re-init gtag